tensorflow/core/grappler/optimizers/generic_layout_optimizer_transposer.cc

```diff
@@ -1567,7 +1567,7 @@ Status StridedSliceTransposer::PermuteMask(TransposeContext* context,
     return errors::InvalidArgument("invalid mask value: ", mask_i);
   }
   int result = 0;
-  for (int i = 0, iter_limit = context->src_to_dst.size(); i < iter_limit;
+  for (int i = 0, end = context->src_to_dst.size(); i < end;
        i++) {
```

Comment:
Should merge in a single line

Reply:
handled in previous PR.

Reply:
Keeping open until rebase removes it from here